features/step_definitions/kata.rb

```diff
@@ -227,3 +227,189 @@
     end
   end
 end
+
+Given /^I run must-gather command$/ do
```

Comment:
@valtair1891 pls document what this step does

Reply:
@valtair1891 if the step is kata specific, then please change the wording of the step as it's too generic presently.